clustered/server/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
   compile group: 'org.terracotta', name: 'offheap-store', version: parent.offheapVersion
   compile group: 'org.slf4j', name: 'slf4j-api', version: parent.slf4jVersion
   provided "org.terracotta:entity-server-api:$parent.entityApiVersion"
+    provided "org.terracotta:standard-cluster-services:$parent.terracottaApisVersion"
```

Comment:
That's because of stupid intellij refusing me to manually format the gradle files.